chaincode/input.go

```diff
@@ -162,14 +162,14 @@ type inputComputePlanTesttuple struct {
 
 type inputLeaderboard struct {
 	ObjectiveKey   string `validate:"omitempty,len=64,hexadecimal" json:"objectiveKey"`
-	AscendingOrder bool   `json:"ascendingOrder,required"`
+	AscendingOrder bool   `validate:"required" json:"ascendingOrder"`
```

Comment:
Be careful the `required` tag  in the lib validator.v9 checks if the field is not zero value : https://godoc.org/gopkg.in/go-playground/validator.v9#hdr-Required
So for a boolean it will enforce the `True` value whereas the json `required` tag will just ensure one value is passed, either `False` or `True`. So let's keep the json one for now.

Reply:
OK, removed `validate:"required"` in both instances.

Also left `json:required` because removing it breaks the tests, although I don’t see that it does anything. What am I missing?